### PR TITLE
Add section on winding order

### PR DESCRIPTION
**Pull request by felixpalmer**

### CODE REVIEW

**Review by alasarr — APPROVED** [review]

**Comment by cholmes** [comment]
I'm out of my depth here, don't think I've ever dug into winding order. Hoping @tschaub or @hobu sounds in on #46 from the geojson perspective.

**Comment by cholmes** [comment]
I'm out of my depth here, don't think I've ever dug into winding order. Hoping @tschaub or @hobu sounds in on #46 from the geojson perspective.

**Comment by Jesus89** [comment]
From this https://github.com/opengeospatial/geoparquet/issues/46#issue-1178648653, this covers the `planar` edges case, but not the `spherical` edges case. I would make this explicit in the spec. Regarding the spherical case, we can leave it undefined or with an explicit definition too.

**Comment by alasarr** [comment]
>From this https://github.com/opengeospatial/geoparquet/issues/46#issue-1178648653, this covers the planar edges case, but not the spherical edges case. I would make this explicit in the spec. Regarding the spherical case, we can leave it undefined or with an explicit definition too.

I think @felixpalmer proposes to force this winding order for both cases

**Comment by cholmes** [comment]
> I think @felixpalmer proposes to force this winding order for both cases

Yes, but I *think* we also need to specify the orientation rule. It probably makes sense to follow [MS SQL](https://docs.microsoft.com/en-us/sql/relational-databases/spatial/spatial-data-types-overview?view=sql-server-ver15#orientation-of-spatial-data), and [Google BigQuery](https://cloud.google.com/bigquery/docs/geospatial-data#polygon_orientation) and 'interpret the side to the left of the line as the content of the ring'.

**Comment by jorisvandenbossche** [comment]
I commented on the issue for the more general discussion (https://github.com/opengeospatial/geoparquet/issues/46).

But specific comment on the text in this PR: can we avoid using the term "right hand rule"? I know the GeoJSON spec uses this, but that term is very confusing, as other sources use "left hand" to specify the same .. (eg the cited bigquery and MS SQL sources)

**Comment by alasarr** [comment]
Thanks @jorisvandenbossche and @cholmes. @felixpalmer I think makes sense to specify that to be more explicit.

File: format-specs/geoparquet.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ The axis order of the coordinates in WKB stored in a geoparquet follows the de f
 (x, y) where x is easting or longitude and y is northing or latitude. This ordering explicitly overrides the axis order as specified in the CRS. 
 This follows the precedent of [GeoPackage](https://geopackage.org), see the [note in their spec](https://www.geopackage.org/spec130/#gpb_spec). 
 
+#### Polygon winding
+
+The winding order of polygons follows the [GeoJSON spec](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6). Polygon rings MUST follow the right-hand rule for orientation (counterclockwise external rings, clockwise internal rings).
```
Comment (tschaub) [inline format-specs/geoparquet.md:110]:
```suggestion
The winding order of polygons follows the [GeoJSON spec](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6). Polygon rings MUST follow the right-hand rule for orientation (counterclockwise external rings, clockwise internal rings).  Traversing vertices of rings in order, the interior of the polygon is on the left.
```

**Comment by tschaub** [comment]
Late to the party, but I agree that this is good to specify.  I think mentioning the "right-hand rule" is ok (as some people may know it), but that it also makes sense to add some more detail (something like "the interior of the polygon is to the left of the boundary as you traverse vertices in order").

Inevitably, consumers of the data will do orientation checks before rendering, topology operations, etc. (as the data gets passed through structures that don't keep track of or require a specific orientation), but I still think it improves things to have the format specify how polygon rings are to be interpreted.

File: format-specs/geoparquet.md (hunk 1 above)
Comment (rouault) [reply format-specs/geoparquet.md:110]: Perhaps it should also be mentioned that this is the opposite convention of a little-known format called Shapefile :-)

**Comment by alasarr** [comment]
@tschaub this is one of the latest PRs of v02.0. I've added your commit suggestion. Could you add your stamp here?

**Review by cholmes — APPROVED** [review]

**Comment by felixpalmer** [comment]
I agree on the clarification here, it makes it more accessible to wider reader base

**Pull request merged**